people/kpis.md

```diff
@@ -0,0 +1,38 @@
+# KPIs for People 
+
+We know our People practices are working when we see the following Key Performance Indicators at healthy levels.
+
+## Healthy team function - well-being and engagement
+
+### Positive team engagement 
+Engagement is a broad category of things that touches on motivation, connection and well-being at work. At least once a year, we complete a team engagement survey. That survey is based on questions from benchmarking organizations, and 2i2c team members can review the detailed survey design and follow up actions on [the People Operations Google Drive](https://drive.google.com/drive/u/2/folders/1JK9f7tF9s4BKW8gNLIDoNLFuV979nkzy). 
+
+Here are our last survey results from April 2025. Our target, what we'd consider "good" scores for each, is 70%. A lower score means we want to take action to improve. 
+Well-being (73%) - are we feeling good? 
+Leadership (76%) - are leaders inspiring and supporting each other and the team?
+Enablement (67%) - does each of us have what they need to be successful?
+Alignment (70%) - do we all understand our roles and the vision for 2i2c?
+Development (55%) - does each person feel like they’re growing?
+Relationships (90%) - are we safe & supportive?
```

Comment:
````suggestion

```{csv-table}
:header: "category", "response", "description"
"Well-being", "73%", "are we feeling good?"
"Leadership", "76%", "are leaders inspiring and supporting each other and the team?"
"Enablement", "67%", "does each of us have what they need to be successful?"
"Alignment", "70%", "do we all understand our roles and the vision for 2i2c?"
"Development", "55%", "does each person feel like they're growing?"
"Relationships", "90%", "are we safe & supportive?"
```
````